### ISSUE TITLE
Windows Defender preventing App from running.

### ISSUE THREAD

**Issue by Hoosky** [rauenzi/BetterDiscordApp#65]
Windows Defender apparently does not like the new update to BetterDiscordApp and blocks off running the installer as it is marked as a "Harmful Program" for some reason (Though I specifically know BandagedBD_Windows.exe is not harmful at all).
I've attempted to find ways of installing it without triggering Windows Defender and what might be causing it.
All currently inconclusive.

I found this strange as the last version of BandagedBD didn't trigger Windows Defender at all.

**Comment by rauenzi** [comment]
Contact me on discord and I will see what I can do.

**Comment by Hoosky** [comment]
Sure,
Where can I find your Discord ID to message you?

**Comment by rauenzi** [comment]
it's plastered all over github Zerebos#7790

**Comment by Hoosky** [comment]
I sent a friend request on Discord so I can DM you.

**Comment by Kallyopae** [comment]
I can't reproduce it either. Win10 1809 with latest cumulative update and win defender definitions at latest version. @rauenzi i have send you discord friend request too. Serket.

**Comment by rauenzi** [comment]
I don't have any pending

**Comment by Kallyopae** [comment]
> I don't have any pending

I have send a request like 1 min ago.

**Comment by Hoosky** [comment]
I have found what was causing the issue.

Apparently Windows Updater was halted for some unknown reason and didn't install the most recent version of Win10 and Windows Defender and never notified me of this.
After updating Win10 and Windows Defender to the most recent version manually it no longer detects BandagedBD as a threat.
I found it strange that being an update behind caused Windows Defender to do this,
But it may have been an actual bug within that Windows Defender version to cause it to false flag safe programs.

Either way,
Upon attempting to install BandagedBD for the first time it did notify me in the installer that I already had the latest version of BandagedBD (even though I didn't)
So I just exited out and retried and was able to successfully install BandagedBD.

Everything is working fine now and it seems this issue has been resolved.
